app/controllers/vacancies_controller.rb

```diff
@@ -1,6 +1,8 @@
 class VacanciesController < ApplicationController
   include ParameterSanitiser
 
+  before_action :set_map_display, only: :index
```

Comment:
Given this is only for one single action, do you reckon it could just live in the action itself?